src/components/Sidebar/AddBranchNode.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC } from 'react';
+import OtherCriteria from './OtherCriteria';
+import ChoiceNode from './ChoiceNode';
+import AddChoiceButton from './AddChoiceButton';
+import McodeCriteria from './McodeCriteria';
+import { State } from 'pathways-model';
+import styles from './Sidebar.module.scss';
+import { Select, MenuItem, FormControl, makeStyles, InputLabel } from '@material-ui/core';
+
+interface BranchNodeProps {
+  currentNode: State;
+  addChoiceNode: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const AddBranchNode: FC<BranchNodeProps> = ({ currentNode, addChoiceNode }) => {
+  // TODO: in PATHWAYS-256 use the properties in the pathway instead?
+  const [source, setSource] = React.useState('');
+  const [critera, setCritera] = React.useState('');
+
+  const handleSourceChange = (event: React.ChangeEvent<{ value: unknown }>): void => {
+    // TODO: in PATHWAYS-256 set the source
+    setSource(event.target.value as string);
+    setCritera('');
+  };
+
+  const handleCriteraChange = (event: React.ChangeEvent<{ value: unknown }>): void => {
```

Comment:
Also remove React here
```suggestion
  const handleCriteraChange = (event: ChangeEvent<{ value: unknown }>): void => {
```